lib/bamboo/mailer.ex

```diff
@@ -84,6 +81,9 @@ defmodule Bamboo.Mailer do
         Use deliver_now/1 to send right away, or deliver_later/1 to send in the background.
         """
       end
+
+      otp_app = Keyword.fetch!(opts, :otp_app)
+      defp config, do: Bamboo.Mailer.parse_opts(__MODULE__, unquote(otp_app))
```

Comment:
Could this be moved below `deliver_later` so it's closer to the function that uses it?